pkg/controller/node/node_controller.go

```diff
@@ -64,8 +67,14 @@ const (
 
 	// masterPoolName is the control plane MachineConfigPool name
 	masterPoolName = "master"
+
+	// kubeletCAFilePath is the expected file path for the kubelet ca
+	kubeletCAFilePath = "/etc/kubernetes/kubelet-ca.crt"
 )
 
+// kubeAPIToKubeletSignerNamePrefixes is the list of subject common names that are regarded as a kube-apiserver-to-kubelet-signer ca certificate
+var kubeAPIToKubeletSignerNamePrefixes = []string{"openshift-kube-apiserver-operator_kube-apiserver-to-kubelet-signer@", "kube-apiserver-to-kubelet-signer"}
```

Comment:
Do we know that these are exclusive prefix or there can be some other prefix too?

Reply:
It's a combination of the namespace,  the secret name, and the timestamp, and the generation happens via the libary-go functions: 
 https://github.com/openshift/library-go/blob/ed9bc958bd8a2fff079d52976806e4e0a8a7c315/pkg/operator/certrotation/signer.go#L132
 
 Which the apiserver-operator has vendored here: 
 https://github.com/openshift/cluster-kube-apiserver-operator/blob/b98a8911dfb04c5c8ccc66dc3bb8c9feebabe4f8/vendor/github.com/openshift/library-go/pkg/operator/certrotation/signer.go#L133
 
 So as long as we don't change the names of the secrets or the namespaces, we should only ever have:
- the original name (non-prefixed/suffixed, not named by library-go) CN=kube-apiserver-to-kubelet-signer
-  the rotated names (prefixed/suffixed, named by library-go) CN=openshift-kube-apiserver-operator_kube-apiserver-to-kubelet-signer@timestamp

Reply:
great, thanks for detail!